rotation.py

```diff
@@ -266,7 +286,9 @@ def main():
     print("")  # Add a newline between rotation output and calendar reminder output.
     log_debug_actions()
     try:
-        add_gcal_reminder(args.production, rotations[-1])  # for next week.
+        add_gcal_reminder(
+            args.production, list(rotations.values())[-1]
```

Comment:
Same: does this need to sorted? Maybe it'd be more explicit to index with the week though, e.g. `rotations[get_week(DATE + timedelta(weeks=1))]`.

Whatever fix is applied here will also need to be applied to `add_gcal_reminder_manually`.